src/App.js

```diff
@@ -1,9 +1,26 @@
 import React, { Component } from 'react';
 import Starships from './components/Starships';
 
-//styles
 import './App.scss';
+
 class App extends Component {
+	state = {
+		counter: 0,
+		loading: true
+	}
+
+
+	componentWillMount(){
+    
+	}
+
+	componentDidMount(){
+	}
+
+	componentWillReceiveProps(){
+
+	}
```

Comment:
Remove the empty componentWillMount, componentDidMount, and componentWillReceiveProps